.github/workflows/test.yml

```diff
@@ -187,6 +187,44 @@ jobs:
           name: "${{ github.sha }}-${{ matrix.part }}-race-output"
           path: ./${{ matrix.part }}-race-output.txt
 
+  test-rosetta:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+      - uses: technote-space/get-diff-action@v4
+        id: git_diff
+        with:
+          PATTERNS: |
+            **/**.go
+            go.mod
+            go.sum
+      - name: setup rosetta CLI
+        # if: env.GIT_DIFF
+        run: |
+          curl -sSfL https://raw.githubusercontent.com/coinbase/rosetta-cli/master/scripts/install.sh | sh -s
+          ./bin/rosetta-cli configuration:create rosetta.conf
+          sed -i 's/Ethereum/app/g' rosetta.conf
+          sed -i 's/Ropsten/network/g' rosetta.conf
```

Comment:
How about caching both the binary and the `rosetta.conf` file? @marbar3778 thoughts?

Reply:
yea good call. Since the binary doesn't change often it makes sense